examples/go.mod

```diff
@@ -0,0 +1,8 @@
+module examples
+
+go 1.14
+
+require (
+	github.com/ably/ably-go v1.2.0-apipreview.3
```

Comment:
Please run `go mod tidy` to apply automatic formatting and other automatic fixes to `go.mod` and `go.sum`.

Also, we're at `v1.2.0-apipreview.4` so this probably should be based on that one.

Reply:
Thanks, will make the change